backend/db/refresh.py

```diff
@@ -46,14 +46,18 @@ def refresh_db(
     """
     local = use_local_db
     print('INFO: Starting database refresh.', flush=True)  # flush: for gh action
-    t0, t0_str = datetime.now(), current_datetime()
+    t0, start_time = datetime.now(), current_datetime()
 
     if is_refresh_active():
         print('INFO: Refresh already in progress. When that process completes, it will restart again. Exiting.')
-        update_db_status_var('new_request_while_refreshing', t0_str, local)
+        update_db_status_var('new_request_while_refreshing', start_time, local)
         return
+    # end_time: Even though in reality the refresh will not end 1 microsecond after the start time, we're setting it
+    # this way because this is the easiest way to make sure that future refreshes will not miss any new data that was
+    # added between when the refresh started and ended.
+    end_time: str = tz_datetime_str(dp.parse(start_time) + timedelta(microseconds=1))
```

Comment:
## `last_refresh_success` fix / redundancy
end_time: Even though in reality the refresh will not end 1 microsecond after the start time, we're setting it this way because this is the easiest way to make sure that future refreshes will not miss any new data that was added between when the refresh started and ended.